image_explorer/public/js/image_explorer.js

```diff
@@ -45,7 +45,7 @@ function ImageExplorerBlock(runtime, element) {
       if (active_feedback) {
         close_feedback();
       }
-
+      $(this).addClass('visited');
```

Comment:
This will only add the ``visited`` class for hotspots opened in the current session. Wouldn't you also want this state to persist?

i.e. If you open a hotspot, it will get the "visited" class, but if you refresh the page, that class will no longer be applied, even though the hotspot is still "visited". The xblock does store a list of visited hotspots, so perhaps it makes sense to use that information to apply the class to previously-visited hotspots.